### PR TITLE
CORE-7186: Use RegEx constants from `corda-api`

### PR DESCRIPTION
**Pull request by vkolomeyko**
Requires: https://github.com/corda/corda-api/pull/666

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -40,7 +40,7 @@ bouncycastleVersion=1.70
 # Corda API libs revision (change in 4th digit indicates a breaking change)
 # Change to 5.0.0.xx-SNAPSHOT to pick up maven local published copy
 #cordaApiVersion=5.0.0.xxx-SNAPSHOT
-cordaApiVersion=5.0.0.423-beta+
+cordaApiVersion=5.0.0.423-alpha-1666884384128
```
Comment (vkolomeyko) [inline gradle.properties]: Will revert this change once `corda-api` changes are integrated.

**Review by conal-smith — APPROVED** [review]
looks good to me

**Review by benmcmahon100 — APPROVED** [review]
